cmake/herbert_Version.cmake

```diff
@@ -1,38 +1,45 @@
+#[=======================================================================[.rst:
+herbert_Version
+-----------------
+
+Build a version release string:
+  <version>[-<date>]-<target>-<matlab>[-<sha>]
+
+ Optional elements are included based on the value of Herbert_RELEASE_TYPE
+ - Date included for "nightly" builds
+ - SHA included for non-"release" builds (i.e. "nightly" or "pull-request"
+
+Variables required by the module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``Herbert_RELEASE_TYPE
```

Comment:
Maybe it would be good to mention here that if `Herbert_RELEASE_TYPE` is empty, not-defined, not `nightly` or not `release` then the default mode is `pull_request` - this is the format we'd want for dev builds as well (with the sha included)

Reply:
added `(default)`